### PR TITLE
Update wyoming-whisper to 2.3.0

### PR DESCRIPTION
**Pull request by duhow**
Bump update from https://github.com/rhasspy/wyoming-faster-whisper/pull/40

## Summary by CodeRabbit

- **New Features**
	- Introduced support for the new `turbo` model for enhanced processing speed.
  
- **Version Updates**
	- Updated `wyoming-whisper` package to version `2.3.0`.
	- Updated `faster-whisper` package to version `1.1.0`.
  
- **Documentation**
	- Changelog updated to reflect new version entries and features.
	- Documentation updated to include the new `turbo` model option and clarifications on configuration options.

### CODE REVIEW

**Comment by synesthesiam** [comment]
Upstream repo has been updated, so the tests can be run now.

**Comment by synesthesiam** [comment]
Release: https://github.com/rhasspy/wyoming-faster-whisper/releases/tag/v2.3.0

File: whisper/build.yaml [hunk 1]
```diff
@@ -6,4 +6,4 @@ codenotary:
   signer: notary@home-assistant.io
   base_image: notary@home-assistant.io
 args:
-  WYOMING_WHISPER_VERSION: 2.2.0
+  WYOMING_WHISPER_VERSION: 2.3.0
```
[the remark this answers is not in the document]
Comment (agners) [reply whisper/build.yaml:9]: This has been addressed meanwhile.

**Review by agners — APPROVED** [review]
LGTM, thanks!

**Pull request merged**